src/packets.cpp

```diff
@@ -433,6 +433,11 @@ AOPacket AOClient::validateIcPacket(AOPacket packet)
     if (incoming_msg == last_message)
         return invalid;
 
+    if (incoming_msg == "" && area->blankposting_allowed == false) {
```

Comment:
incoming_msg.trimmed() == ""

https://doc.qt.io/qt-5/qstring.html#trimmed

otherwise this will only reject empty messages